pkg/util/utils.go

```diff
@@ -30,6 +30,9 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// ProxyVars is being used by different packages of podman
+var ProxyVars = [...]string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY", "http_proxy", "https_proxy", "no_proxy", "ftp_proxy", "FTP_PROXY"}
```

Comment:
Can you do this as a map[string]bool and then use it in both places.

Reply:
Yes that is better. I'll amend this.

Reply:
@rhatdan I am not sure what would be advantage though cause for `podman` we have to allow all of them anyways. Is intention to make them configurable ?

Reply:
No just sharing with the other code, which needs a map.

Reply:
@rhatdan Oh i see. Only `buildah` is using a map for this do we intend to share this with `buildah` ? I don't see anywhere else

Although `c/common` is using a similar line https://github.com/containers/common/blob/main/pkg/config/config.go#L900
We could make this variable `Global` and use it.

The only small issue is then this PR would need vendor changes as well. But its not a big deal. Let me know what you feel fits best here.